bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -519,20 +553,15 @@ public void mouseClicked(final MouseEvent e) {
         dragSource = new TelescopePosTableDragSource(this);
     }
 
-    public void telescopePosUpdate(final WatchablePos tp) {
+    @Override public void telescopePosUpdate(final WatchablePos tp) {
         final int index = getSelectedRow();
         _resetTable(_env);
 
         // Restore the selection without firing new selection events.
-        if ((index >= 0) && (index < getRowCount())) {
-            final boolean ignore = _ignoreSelection;
-            try {
-                _ignoreSelection = true;
-                _setSelectedRow(index);
-            } finally {
-                _ignoreSelection = ignore;
-            }
-        }
+        final boolean tmp = _ignoreSelection;
+        _ignoreSelection = true;
+        _setSelectedRow(index);
```

Comment:
Was there no reason for wrapping this in a `try`?  If it should throw an exception for any reason the `_ignoreSelection` flag will remain `true` and nothing will work.